client/src/components/Description.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components'
+
+class Description extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = (
+            { show: false }
+        )
+        this.show = this.show.bind(this);
+    }
+    show(){
+        this.setState({
+            show: !this.state.show
+        })
+    }
+    render() {
+        if (this.props.description===undefined){
+            return <div>
```

Comment:
Perhaps these divs can be aligned either vertically or on the same line?

Reply:
resolved :)